frontend/src/pages/QueryBuilderChooser/QueryBuilderChooser.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { FC, useState, useEffect } from 'react';
+import * as localForage from 'localforage';
+import { localForageKeys, api } from '../../utils';
+import { Alert, Button, Modal } from 'react-bootstrap';
+import { RouteComponentProps } from 'react-router-dom';
+import { ICheckData, IRadio } from '../../components/IRadio';
+import AdvancedQueryBuilder from '../AdvancedQueryBuilder/AdvancedQueryBuilder';
+import QueryBuilder from '../QueryBuilder/QueryBuilder';
+import { CheckBox } from '../../components/CheckBox';
+import axios from 'axios';
+
+type SelectedQueryBuilder = 'basic' | 'advanced';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+const QueryBuilderChooser: FC<RouteComponentProps> = (props: RouteComponentProps<{}>) => {
+  const [showModal, setShowModal] = useState(true);
+  const [selectedOption, setSelectedOption] = useState<SelectedQueryBuilder>('advanced');
+  const [checked, setChecked] = useState(false);
+  const [choice, setChoice] = useState<string>();
+  const [redirectPage, setRedirectPage] = useState('');
```

Comment:
This variable looks redundant, I don't see it's importance. Why did you add it? I feel like the variable selectedOption is enough, and the redirectPage variable is just a repetition of the same